src/common/date-utils.ts

```diff
@@ -30,3 +30,14 @@ export function displayDate(txDate: string): string {
   }
   return date.format('MMM Do, YYYY');
 }
+
+export function isoDateToLocalDateSafe(isoDate: string) {
+  let result;
+  try {
+    result = isoDateToLocalDate(isoDate);
+  } catch (e) {
+    console.log(isoDate);
+    console.log(e);
+  }
+  return result;
+}
```

Comment:
```suggestion
export function isoDateToLocalDateSafe(isoDate: string) {
  try {
    return isoDateToLocalDate(isoDate);
  } catch (e) {
    console.log(isoDate);
    console.log(e);
  }
}
```

no need to redeclare right?